hangman.rb

```diff
@@ -0,0 +1,124 @@
+class Hangman
+
+  def initialize
+    @now_playing = true
+    @count = 0
+
+    # Board Display
+    @row1 = "|     _________"
+    @row2 = "|     |/      |"
+    @row3 = "|     |      "
+    @row4 = "|     |      "
+    @row5 = "|     |       "
+    @row6 = "|     |      "
+    @row7 = "|     |"
+    @row8 = "| ____|_____"
+
```

Comment:
I like how you clearly set up this board display section with everything aligned and commented.